*** /tmp/,RCSt1a06681	Mon Feb 10 16:03:22 1992
--- Imakefile	Fri Jan 10 09:27:15 1992
***************
*** 6,12 ****
--- 6,16 ----
  LIBJPEG = $(JPEGDIR)/libjpeg.a
  JPEGINCLUDE = -I$(JPEGDIR)
  
+ TIFFDIR = libtiff
+ LIBTIFF = $(TIFFDIR)/libtiff.a
+ TIFFINCLUDE = -I$(TIFFDIR)
  
+ 
  /* if you are running under DXWM, I pity you.  XV doesn't work correctly
   * under DXWM.  You should probably be running MWM.  However, if such is
   * not a valid option for you, try uncommenting the following line.  The
***************
*** 104,112 ****
  
  
  DEFINES= $(SCO) $(UNIX) $(NEED_DIRENT) $(VPRINTF) $(TIMERS) $(ATT) \
! 	$(USEMEM) $(HPUX7) $(JPEG) $(JPEGINCLUDE) $(DXWM) $(UMAX)
  
! SYS_LIBRARIES=	-lX11_s $(LIBJPEG) -lm
  
  BITMAPS = bitmaps/grasp bitmaps/penn bitmaps/down bitmaps/down1 \
  	  bitmaps/up bitmaps/up1 bitmaps/scrlgray bitmaps/gray50 \
--- 108,117 ----
  
  
  DEFINES= $(SCO) $(UNIX) $(NEED_DIRENT) $(VPRINTF) $(TIMERS) $(ATT) \
! 	$(USEMEM) $(HPUX7) $(JPEG) $(JPEGINCLUDE) $(TIFFINCLUDE) \
! 	$(DXWM) $(UMAX)
  
! SYS_LIBRARIES=	-lX11_s $(LIBJPEG) $(LIBTIFF) -lm
  
  BITMAPS = bitmaps/grasp bitmaps/penn bitmaps/down bitmaps/down1 \
  	  bitmaps/up bitmaps/up1 bitmaps/scrlgray bitmaps/gray50 \
***************
*** 131,144 ****
  	xvxbm.c xvsunras.c xvjpeg.c vprintf.c
  
  SRCS1 =	xv.c xvevent.c xvroot.c xvmisc.c xvimage.c xvcolor.c xvsmooth.c \
! 	xv24to8.c xvgif.c xvpm.c xvinfo.c xvctrl.c xvscrl.c \
! 	xvgifwr.c xvdir.c xvbutt.c xvpbm.c xvxbm.c xvgam.c \
  	xvdial.c xvgraf.c xvsunras.c xvjpeg.c xvps.c xvpopup.c xvdflt.c \
  	vprintf.c 
  
  OBJS1 =	xv.o xvevent.o xvroot.o xvmisc.o xvimage.o xvcolor.o xvsmooth.o \
! 	xv24to8.o xvgif.o xvpm.o xvinfo.o xvctrl.o xvscrl.o \
! 	xvgifwr.o xvdir.o xvbutt.o xvpbm.o xvxbm.o xvgam.o \
  	xvdial.o xvgraf.o xvsunras.o xvjpeg.o xvps.o xvpopup.o xvdflt.o \
  	vprintf.o 
  
--- 136,149 ----
  	xvxbm.c xvsunras.c xvjpeg.c vprintf.c
  
  SRCS1 =	xv.c xvevent.c xvroot.c xvmisc.c xvimage.c xvcolor.c xvsmooth.c \
! 	xv24to8.c xvgif.c xvtiff.c xvpm.c xvinfo.c xvctrl.c xvscrl.c \
! 	xvgifwr.c xvtiffwr.c xvdir.c xvbutt.c xvpbm.c xvxbm.c xvgam.c \
  	xvdial.c xvgraf.c xvsunras.c xvjpeg.c xvps.c xvpopup.c xvdflt.c \
  	vprintf.c 
  
  OBJS1 =	xv.o xvevent.o xvroot.o xvmisc.o xvimage.o xvcolor.o xvsmooth.o \
! 	xv24to8.o xvgif.o xvtiff.o xvpm.o xvinfo.o xvctrl.o xvscrl.o \
! 	xvgifwr.o xvtiffwr.o xvdir.o xvbutt.o xvpbm.o xvxbm.o xvgam.o \
  	xvdial.o xvgraf.o xvsunras.o xvjpeg.o xvps.o xvpopup.o xvdflt.o \
  	vprintf.o 
  
